src/assets/translations/knb.json

```diff
@@ -32819,7 +32819,7 @@
                   "verse": "23"
                 },
                 {
-                  "text": "Ezért azt mondták egymásnak: »Ezt ne vágjuk szét, inkább vessünk rá sorsot, kié legyen!« Ez azért történt, hogy beteljesedjék az Írás, amely így szól: A katonák tehát ezt tették.»Elosztották maguk között ruháimat, és köntösömre sorsot vetettek«.",
+                  "text": "Ezért azt mondták egymásnak: »Ezt ne vágjuk szét, inkább vessünk rá sorsot, kié legyen!« Ez azért történt, hogy beteljesedjék az Írás, amely így szól: »Elosztották maguk között ruháimat, és köntösömre sorsot vetettek.« A katonák tehát ezt tették.",
```

Comment:
This is a critical change. I propose to change the KNB biblical text, as it contains a clear error.
In case this proposal is accepted, the following setup task will needed, to document this decision:
https://github.com/anwolosz/synopsis/issues/109

Reply:
Explained in #109. Accepted.